### ISSUE TITLE
WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.menu-item-deny)

### ISSUE THREAD

**Issue by BorisGuan** [BorisGuan/MDSIssuesRepro#12]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Master Data Services](http://eims-mds-01/MDS/security/common/HierarchyMembers.aspx?PID=bf0cef35-6e21-4835-980e-aaf1ebd91da3&PTID=1&m=2&MID=00000000-0000-0000-0000-000000000000)
#### Element path
.menu-item-deny
#### Snippet
`<li class="menu-item-deny mds-icon-permission-deny disabled" permission="128">Deny</li>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.62 (foreground color: #808080, background color: #f5f5f5, font size: 8.3pt (11px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 88.0.4324.104
This accessibility issue was found using Accessibility Insights for Web 2.23.0 (axe-core 4.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
